lib/smart_app_launch/endpoints/mock_smart_server/smart_response_creation.rb

```diff
@@ -0,0 +1,266 @@
+require_relative '../../tags'
+require_relative '../mock_smart_server'
+require_relative '../../client_suite/oidc_jwks'
+
+module SMARTAppLaunch
+  module MockSMARTServer
+    module SMARTResponseCreation
```

Comment:
I don't understand the reasoning behind lumping the code for the responses to the authorization, introspection, and token endpoints into this file rather than putting it in the relevant endpoints..

Reply:
It is needed to make supporting SMART and UDAP in other test kits easier.  I previously had all of the endpoint logic in a single class, but then was going to need to write a lot of glue code to get them to work together. With this approach, I'm able to mix both sets of methods in and have simple glue code that delegates to one set or the other, see [carin for example](https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/client_auth/lib/carin_for_blue_button_test_kit/client/v2.0.0/endpoints/token_endpoint.rb#L11).  Is there a different way you would approach this?

I made some changes to separate out token and authorization code, but I'm not sure it was worth it.

Reply:
separated in [977486a](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/96/commits/977486ab23d76914af147d86e84c5935f66533b3)